test/regress/conf/coordinator.yaml

```diff
@@ -1,5 +1,7 @@
+log_level: info
 host: regress_coordinator
+dbname: spqr-console
```

Comment:
lets remove this and enforce in code instead. 
there no reason for user to think about usr/db when it connects to coordinator. 
It makes a lot of sense in router case, since this params is actually used for routing there.